draft-ietf-privacypass-batched-tokens.md

```diff
@@ -75,10 +77,17 @@ ways:
 1. Improving server and client issuance efficiency by amortizing the cost of the
    VOPRF proof generation and verification, respectively.
 
-# Client-to-Issuer Request {#client-to-issuer-request}
+For the Publicly Verifiable Token issuance protocol, it allows for a single TokenRequest
+to be sent that encompasses multiple tokens. The cost remains linear.
```

Comment:
This is also true for the arbitrary batched issuance variant with private tokens